clients/cloud/java-springboot/src/main/resources/application.properties

```diff
@@ -1,26 +1,24 @@
 # topic config
-io.confluent.developer.config.topic.name=test
+io.confluent.developer.config.topic.name=test2
```

Comment:
It's not clear if this was intentional

Reply:
I determined that this wasn't a likely intentional change (based on the remnant topic `test` in docs), so I reverted this bit.